src/librustc_target/spec/wasm32_unknown_wasi.rs

```diff
@@ -0,0 +1,113 @@
+//! The `wasm32-unknown-wasi` target is a new and still (as of March 2019)
+//! experimental target. The definition in this file is likely to be tweaked
+//! over time and shouldn't be relied on too much.
+//!
+//! The `wasi` target is a proposal to define a standardized set of syscalls
+//! that WebAssembly files can interoperate with. This set of syscalls is
+//! intended to empower WebAssembly binaries with native capabilities such as
+//! filesystem access, network access, etc.
+//!
+//! FIXME: insert link to proposal here
```

Comment:
☝️

```suggestion
https://github.com/CraneStation/wasmtime-wasi/blob/wasi/docs/WASI-proposed-CG-subgroup-charter.md
```

or maybe just wasi.dev?